.pylintrc

```diff
@@ -429,13 +429,6 @@ valid-metaclass-classmethod-first-arg=mcs
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception"
-overgeneral-exceptions=StandardError,
-                       Exception,
-                       BaseException
-
-#######
-
-# https://github.com/edaniszewski/pylint-quotes#configuration
-string-quote=single
-triple-quote=double
-docstring-quote=double
```

Comment:
We want to keep these behaviors : )

Reply:
@Michaelvll Use Pylint with this configuration instead of `pylint-quotes` should work well for most cases in the existing repo:

```toml
[tool.pylint]
check-quote-consistency = true
load-plugins = ["pylint.extensions.docstyle"]
```

The `pylint-quotes` is deprecated due to no updates in 3 years and incompatibility with `pylint >= 3.0`.